js/gestionPresupuesto.js

```diff
@@ -1,18 +1,42 @@
 // TODO: Crear las funciones, objetos y variables indicadas en el enunciado
 
 // TODO: Variable global
+var presupuesto = 0;
 
-
-function actualizarPresupuesto() {
-    // TODO
+function actualizarPresupuesto(valor) {
+    if (valor >= 0) {
+        presupuesto = valor;
+        return valor;
+    }
+    console.log("Error valor negativo: " + valor);
+    return -1;
 }
 
 function mostrarPresupuesto() {
-    // TODO
+    return `Tu presupuesto actual es de ${presupuesto} €`;
 }
 
-function CrearGasto() {
-    // TODO
+function CrearGasto(descripcion, valor) {
+    if (typeof valor !== "number" || valor < 0) {
+        valor = 0;
+    }
+
+    return {
```

Comment:
Las funciones constructoras no suelen llevar return. Puedes hacer referencia a los campos a través de `this` directamente.